README.md

```diff
@@ -31,9 +31,11 @@ implementation 'com.grafana:grafana-opentelemetry-starter:1.1.0'
 
 The next step depends on your logging framework - logback and log4j2 are supported:
```

Comment:
This statement seems out of place to me.  Perhaps better under Logging.

### Logging

To set up logging, select one of the following frameworks.

#### Logback

Reply:
Done on #23